documentation/deliberately_vulnerable_flask_app/integration_test_utils.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
We really need to figure out a way to share these integration test utilities. We are just spreading them across the codebase, leading to a lot of duplicates. Unfortunately, I don't have a good solution for that.

Since this is not used by anything else that `run_integration_test.py`, could we just move it in a single file?

Reply:
@arthaud yes I plan to share this code across Opensource and our internal integration tests. What do you think should be a good place for this to live? Something like `pyre-check/tools/integration_tests` ? cc @gbleaney who mentioned a possible refactoring

Reply:
@abishekvashok Let's move this and the script launching the integration tests in `pyre-check/tools/integration_tests` :) We may find a better place in future but right now this should work